sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/query.sql

```diff
@@ -0,0 +1,84 @@
+SELECT
+  submission_date,
+  app_version,
+  os,
+  channel,
+  locale,
+  browser_version,
```

Comment:
Should we use standard naming?
```suggestion
  app_version
```

Reply:
Good catch! I was having both. I will remove `browser_version`